indico/htdocs/js/indico/modules/events/management.js

```diff
@@ -110,6 +110,20 @@
         $('.js-reset-role-filter').toggleClass('disabled', isInitialState);
     }
 
+    function initTooltip() {
```

Comment:
I would move this inside `setupEventPersonsList` since it's related to that (the whole management.js is a bit ugly/big, but not worth splitting/refactoring it now)